sdk/metric/sdk.go

```diff
@@ -592,7 +663,12 @@ func (m *SDK) checkpoint(ctx context.Context, descriptor *export.Descriptor, rec
 		return 0
 	}
 	recorder.Checkpoint(ctx, descriptor)
-	exportLabels := export.NewLabels(labels.sorted, labels.encoded, m.labelEncoder)
+
+	// TODO Labels are encoded once per collection interval,
```

Comment:
Same here about filing an issue about it?

Reply:
Will do.

Reply:
#539